qttime.cc

```diff
@@ -0,0 +1,20 @@
+#include "qttime.h"
+
+int time_get_time(lua_State *L) {
+  QDateTime time(QDateTime::currentDateTime());
+  const char* format_raw = luaL_checkstring(L, 1);
+  QString format(format_raw);
+  QString time_string(time.toString(format));
+  const char* time_string_raw = qPrintable(time_string);
```

Comment:
Could we use `qPrintable( time.toString(format_raw))`?
It looks shorter and clear without adding new variables at all.